### ISSUE TITLE
Open to adding a react-native flag to package.json?

### ISSUE THREAD

**Issue by chmac** [ThaUnknown/uint8-util#1]
React Native ends up with `browser.js` which doesn't work because `TextDecoder` doesn't exist.

Adding `"react-native": "index.js"` to `package.json` fixes it for me locally. I can create a PR if that's helpful.

**Comment by chmac** [comment]
Oh, and I forgot to say, thanks for this package by the way. It's a real time saver to have these utils ready to go, without having to think about optimising, etc. 🧡

**Comment by ThaUnknown** [comment]
using index suggests using buffer, which react native doesn't have either, and is polyfilled in the same way you should polyfill textdecoder if its missing

**Comment by ThaUnknown** [comment]
wait no, this package doesn't even have an index.js.......... what the hell did you do

**Comment by chmac** [comment]
Interesting, I just copied the `index.js` from the `package.json` `main` field. But now I realise I was looking at `util.js`, which doesn't seem to have any external dependencies. That's why I chose this package actually!

**Comment by chmac** [comment]
So turns out adding `  "react-native": "util.js",` works as well.

Importing from `uint8-util/util` also works, but TypeScript can't figure out the types then.

**Comment by chmac** [comment]
Right, now I realise that the function I actually want `arr2hex` is in `util.js`, but there's a bunch of other stuff in the package which is not. So importing `util.js` only works for me, but won't work for the other functions...

Maybe it's faster if I just copy your `util.js` into our project and take the bits I need. I guess figuring out all the intricacies of making it play nice with React Native might be overkill.
